server_config_parser_test.cc

```diff
@@ -2,37 +2,38 @@
 #include "server_config_parser.h"
 
 
-
 class PortParseTest : public ::testing::Test {
 protected:
+
     bool getPortNumber(std::string config_contents) {
-       	
        	int fd;
 		char name[] = "/tmp/fileXXXXXX";
-		fd = mkstemp(name); // creates a temporary file using name, replacing the last 6 X's 
+		
```

Comment:
Our indentation is off here but it is okay since we'll probably be changing this whole config parser to be more generic.